Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -253,9 +345,529 @@ def fetch_incidents(client):
     return incidents
 
 
+def parse_limit_and_offset_values(limit: str, offset: str) -> Tuple[int, int]:
+    """
+    Parse the offset and limit parameters to integers and verify that the offset/limit are valid.
+
+    Args:
+        limit (str): limit argument.
+        offset (str): offset argument.
+
+    Raises:
+        ValueError: in case the offset/limit values are invalid
+
+    Returns:
+        Tuple[int, int]: parsed offset and parsed limit
+    """
+    limit, offset = arg_to_number(arg=limit, arg_name="limit"), arg_to_number(arg=offset, arg_name="offset")
+
+    try:
+        assert offset is not None and offset >= 0
```

Comment:
i think it would be better to provide the error message as the second arg to `assert` instead of the `try-except`:
```python
assert offset is not None and offset >= 0, "Offset is invalid, offset scope >= 0"
```

Reply:
i agree with you, ill fix